### PR TITLE
New Pantheon Search with Solr 8 for Drupal 9 page

### PR DESCRIPTION
**Pull request by carolyn-shannon**
## Summary

**[Solr 8 for Drupal 9](https://pantheon.io/docs/guides/solr-drupal/solr-drupal-9)** -  Expanded [Solr for Drupal](https://pantheon.io/docs/guides/solr-drupal) guide consolidates instructions for working with Pantheon Search and Solr across Drupal versions. New guide [Solr 8 for Drupal 9](https://pantheon.io/docs/guides/solr-drupal/solr-drupal-9) provides detailed information on using Pantheon Search with Solr 8 on Drupal 9 with Integrated Composer.

## Remaining Work

The following changes still need to be completed:

- [x] Add Early Access flag
- [x] Section Overview: add overview details
- [ ] Section D9: Installation 
- [ ] Section D9: How to index your site
- [ ] Section D9: Troubleshooting
- [ ] Section: Scope of support
- [ ] Technical Review by Product Team
- [ ] copy review
--------------------------------------------------
## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Add redirects for merged pages `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by carolyn-shannon** [comment]
Add indexing steps: 
- make changes to Drupal’s database schema/fields/content types
- update Solr config files on the Solr server in site’s /core-{SITE_ID}/ folder related to those field changes
- re-index content

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6481](https://pr-6481--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `55.00000000000001` | `53` | `-2` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6481` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/28038/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/28038/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6481--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/solr-drupal-9.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+---
+[Apache Solr](/solr) is a system for indexing and searching site content.
+
+<Alert title="Important Note" type="info">
+
+**Pantheon Search with Solr 8** derives from Solr and can perform full-text content searching in a single language.
+
+<Partial file="solr-version.md" />
+
+If your search needs include geospatial search, emojis, or multilingual search, consider [OpenSolr](/opensolr) or another alternative search.
```
Comment (zealws) [inline source/content/solr-drupal-9.md:18]: > If your search needs **to** include ...

File: source/content/solr-drupal-9.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+## Before You Begin
+
+Be sure that you:
+
+- Enable Solr in the Pantheon Site Dashboard: **Settings** > **Add Ons** > **Apache Solr Index Server: Add**.
+
+- Install [Composer](https://getcomposer.org/)
+
+- Create a Drupal 9 with Integrated Composer managed site on Pantheon following the [Drupal 9](/drupal-9) guide, or migrate to Drupal 9 with Integrated Composer using the (Drupal 9 Migration Guide)[/guides/drupal-9-migration].
```
Comment (zealws) [inline source/content/solr-drupal-9.md:32]: > Create a Drupal 9 **site** with ...

File: source/content/solr-drupal-9.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+To configure the connection with Pantheon, set the [connection mode](/sftp/#sftp-mode) to SFTP and complete the following steps.
+
+### Enable Modules
+
+Enable the Search API Pantheon module via the [Drupal interface](https://www.drupal.org/docs/8/extending-drupal-8/installing-contributed-modules-find-import-enable-configure-drupal-8#enable_your_mod). When prompted, click **Continue** to enable the [Search API](https://www.drupal.org/project/search_api) and [Search API Solr](https://www.drupal.org/project/search_api_solr) modules:
+
+### Disable Drupal Core's Search Module (Optional)
+
+If you are using Search API, then you probably will not be using Drupal Core's Search module. Uninstall the Search core module from `/admin/modules/uninstall` to avoid confusion in further configuration steps.
```
Comment (zealws) [inline source/content/solr-drupal-9.md:87]: > If you are using Search API **Pantheon**, then you ...

File: source/content/solr-drupal-9.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+- Server name: Pantheon
+
+- Backend: Solr
+
+- Solr Connector: Pantheon
+
+- Schema file: `modules/search_api_solr/solr-conf/4.x/schema.xml` (recommended)
+
+You can name the server anything you want but using something like "Pantheon" is a good way to remember where the connection goes. The Search API module provides schema files for each version of Solr (4, 5, and 6). You can customize schema files by copying these examples to your own custom module and editing them. If you are just getting started, we recommend selecting the file for Solr 4.
```
Comment (zealws) [inline source/content/solr-drupal-9.md:101]: I don't think we want them solecting the schema file for Solr 4 anymore. But we should test the updated schema to be sure.

File: source/content/solr-drupal-9.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+The following code changes are required before Solr can be safely uninstalled and disabled:
+
+<Partial file="remove-addons/d8-solr.md" />
+
+## Troubleshooting
+
+### Solr Verification Check
+
+Because we are posting the 4.x schema to a 3.x Solr instance, the schema verification check can fail and prevent indexing. You can disable the schema check by checking the **Skip schema verification** box in the UI, or pulling [this patch](https://www.drupal.org/project/search_api_solr/issues/3037213#comment-12996162) to the module.
```
Comment (zealws) [inline source/content/solr-drupal-9.md:143]: This is out of date, and needs to be changed.

File: source/content/solr-drupal-9.md (hunk 2 above)
Comment (carolyn-shannon) [reply source/content/solr-drupal-9.md]: Having a D9 site w Integrated Composer is a requirement for using this doc, and has been specified in the "Will this guide work for your site?" section.

**Pull request merged**